nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -18,6 +18,7 @@
 
 import torch
 from omegaconf import DictConfig, ListConfig
+from pytorch_lightning.loops.fetchers import _DataFetcherWrapper
```

Comment:
Just noting here we're using an `_` class. Do we need to ask PTL for an API for this?